queries/styled/highlights.scm

```diff
@@ -0,0 +1,4 @@
+; inherits: css
+
+(plain_value) @identifier
+(js_expression) @identifier
```

Comment:
```suggestion
(plain_value) @variable
(js_expression) @variable
```

See https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights for allowed capture names.

Reply:
Thanks, updated